# Implement first (simplest) solution to
#   pytestFailures/solutions/ownerDocument.context.md
--- a/plasTeX/DOM/__init__.py
+++ b/plasTeX/DOM/__init__.py
@@ -1071,6 +1071,14 @@

         """
         charsubs = charsubs or []
+        if not getattr(self, "doCharSubs", True):
+            charsubs = []
+        else:
+            try:
+                if self.ownerDocument.context.isMathMode:
+                   charsubs = []
+            except AttributeError :
+                pass

         if self.hasAttributes():
             for key, value in self.attributes.items():
